bin/propolis-server/src/lib/migrate/source.rs

```diff
@@ -83,16 +200,74 @@ struct SourceProtocol<T: AsyncRead + AsyncWrite + Unpin + Send> {
 
     /// Transport to the destination Instance.
     conn: WebSocketStream<T>,
+
+    /// Guest page table dirty bits to restore in the event of a migration
```

Comment:
It would be useful to mention that dirty bits are accumulated across all phases of memory transfer.